src/main/java/com/shmkane/sellstick/RandomString.java

```diff
@@ -4,11 +4,11 @@
 
 import net.md_5.bungee.api.ChatColor;

```

Comment:
You didn't do this, but I wonder if this should use Bukkit ChatColor instead

Reply:
What I mean by "You didn't do this" is I probably mis-imported at some point

Reply:
Yeah probably, will look in to